stories/DataTable/Basic/BasicSelect.stories.js

```diff
@@ -34,6 +34,7 @@ const BasicSelectable = () => (
     columns={columns}
     data={data}
     selectableRows
+    selectableRowsDisabledField="notSelectable"
```

Comment:
no need for this

Reply:
okay, so you meant `disabled` can be default prop value for `selectableRowsDisabledField` instead of null as of now?